app/views/layouts/iframe.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "_generic.njk" %}
+
+{% set htmlClasses = "app-no-canvas-background" %}
+
+{% block pageTitle %}GOV.UK Frontend{% endblock %}
+
+{# Turn the header and footer off #}
+{% block header %}
+  {% include "../partials/banner.njk" %}
+{% endblock %}
+{% block footer %}{% endblock %}
+
+{% block main %}
+  {% block beforeContent %}{% endblock %}
+  {% block content %}{% endblock %}
+{% endblock %}
+
+{% block bodyEnd %}
+  {{ super() }}
+  {% block scripts %}{% endblock %}
+{% endblock %}
```

Comment:
Would it make sense to call this file `component-iframe.njk`? We already have `component-preview.njk`.

Reply:
@hannalaakso Done